shared/signup/phone-number/phone-input.tsx

```diff
@@ -238,6 +238,7 @@ type Props = {
   defaultCountry?: string
```

Comment:
does this file want to move to common adapters?

Reply:
ya

Reply:
done